src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.TaskType;
+import duke.task.ToDo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage that loads tasks from the specified file and saves tasks to the same file.
+ */
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads tasks from the specified file path. Create the folder if necessary.
+     * @throws FileNotFoundException
+     */
+    public ArrayList<Task> load() throws FileNotFoundException {
```

Comment:
Perhaps instead of using `load` for the name, you can use `loadFiles` for better clarity?